api/src/main/java/org/openmrs/api/ConceptService.java

```diff
@@ -1970,5 +1971,43 @@ public List<ConceptSearchResult> getOrderableConcepts(String phrase, List<Locale
 	 */
 	@Authorized(PrivilegeConstants.GET_CONCEPTS)
 	boolean hasAnyConceptAttribute(ConceptAttributeType conceptAttributeType);
+
+	/**
+	 * Creates or updates the given {@link ConceptReferenceRange} in the database
+	 *
+	 * @param conceptReferenceRange ConceptReferenceRange to save
+	 * @return the created ConceptReferenceRange
+	 * 
+	 * @since 2.7.0
+	 * 
+	 * <strong>Should</strong> create a new concept reference range
+	 * <strong>Should</strong> edit an existing concept reference range
+	 */
+	@Authorized(PrivilegeConstants.MANAGE_CONCEPT_REFERENCE_RANGES)
```

Comment:
We should not create a new privilege for this. Whoever is able to manage concepts should also be able to manage concept reference ranges.